cmd/start.go

```diff
@@ -55,36 +56,39 @@ to quickly create a Cobra application.`,
 
 		var wg sync.WaitGroup
 		var serviceConfig config.ServiceConfig
-		secret := awsutils.GetSecret(os.Getenv("CONFIG"), os.Getenv("REGION"))
+		secret := awsutils.GetSecret(os.Getenv("CONFIG"), os.Getenv("REGION"), logger)
 		err := json.Unmarshal([]byte(secret), &serviceConfig)
 		if err != nil {
-			logger.Fatal(err)
+			logger.Fatal(err.Error())
 		}
 		if cliConfig.Opensource.Enabled {
+			openSourceLogger := setupLogging()
```

Comment:
Why multiple setupLogging? You can have single instance and use it

Reply:
Done